app/src/main/java/org/p2p/wallet/bridge/api/response/BridgeBundleResponse.kt

```diff
@@ -32,4 +33,6 @@ class BridgeBundleResponse(
     val status: ClaimStatus? = null,
     @SerializedName("claim_key")
     val claimKey: String? = null,
+    @SerializedName("created")
+    val dateCreated: MillisSinceEpoch
```

Comment:
dateCreated is not millis

Reply:
and what do you suggest?

Reply:
making just Long and maybe add sec to name

Reply:
fixed